docker-compose.test.yml

```
@@ -3,13 +3,18 @@
       context: .
       dockerfile: Dockerfile
-    command: pnpm run start-prod
+    command: sh -c "./node_modules/.bin/next build && ./node_modules/.bin/next-export-optimize-images && ./node_modules/.bin/serve -s out -p 3001"
     restart: unless-stopped
     volumes:
       - .:/app
       - /app/node_modules
     ports:
       - '3001:3001'
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://localhost:3001" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)